packages/react-rich-text-renderer/.babelrc.js

```diff
@@ -1 +1,8 @@
-module.exports = require("../../.babel.react")({ path: __dirname });
+const defaults = require("../../.babel.react")({
+    path: __dirname
+});
+
+module.exports = {
+    ...defaults,
+    plugins: [...defaults.plugins, "@babel/plugin-transform-modules-commonjs"]
```

Comment:
@endymion1818 This babel plugin dependency needs to be added to the `package.json` for that respective package.